### PR TITLE
6 show reviews

### LINKED ISSUE #6: Show reviews on Restaurant list page
As a User
So I can decide which restaurant to visit
I want to see all reviews on the list of restaurants

### PR DESCRIPTION
**Pull request by AttyC**
Closes #6

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -15,6 +15,13 @@ def create
     redirect_to @restaurant
   end
 
+  def destroy
+    @restaurant = Restaurant.find(params[:restaurant_id])
+    @review = Review.find(params[:id])
+    @review.destroy
+    redirect_to restaurant_path(@restaurant)
```
Comment (rcvink) [inline app/controllers/reviews_controller.rb:22]: Cool beans

**Review by ghost — APPROVED** [review]
:shipit:

File: spec/features/display_reviews_spec.rb [hunk 2]
```diff
@@ -10,4 +10,22 @@
...
+  feature 'Delete review' do
+    scenario 'when on resturant show page' do
+      create_restaurant('Testaurant', 'Delicious')
+      create_review(2, 'really good!!')
+      create_review(5, 'terrible rat problem!')
+      click_on 'Remove Review 1'
+      within('div#reviews li:first-child') {expect(page).not_to have_content('really good!!')}
+    end
+  end
 end
```
Comment (rcvink) [inline spec/features/display_reviews_spec.rb:31]: :wave:

**Review by rcvink — APPROVED** [review]
Top drawer stuff

**Pull request merged**